project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -0,0 +1,60 @@
+# Stage 2: Incubation
+
+*As a project intending to move to stage 2 of the OpenRail incubation process, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+
+*Copy this template and fill in your answers to the questions in the sections below.*
+
+## Health metrics
+
+*How to assess project health depends on the context of the concrete project. Please answer the questions below and add any additional explanations or data you see as useful to show that the project has a healthy community and development activity.*
+
+### How many committers does the project have?
+
+
+### How many different organizations are contributing to the project?
+
+
+### How much is the project dependent on individual contributors?
+
+*How high is the risk to a project should the most active people leave? This is called the [Contributor Absence Factor](https://chaoss.community/kb/metric-contributor-absence-factor/), sometimes also referred to as bus factor.*
+
+
+### How active is the project?
+
+*Indicate activity by number of commits, number of issues closed, pull request merged, etc.*
+
+
+## Open governance
+
+### What is the release process of the project?
+
+*Indicate documentation of the release process, how changes are communicated, versioning schemes, signing of release artifacts, etc.*
+
+
+### How are decisions taken in the project?
+
+*Indicate documentation of the project governance.*
```

Comment:
Is the `GOVERNANCE.md` file is enough?
What decision? (roadmap, merge PR, technical design...)
We should add precision for this question.

Reply:
I kind of like the idea that the GOVERNANCE.md file should be enough. Should we then require that in addition to the formal high-level decision making process it also includes information about how more technical decisions are taken, such as merging PRs, design, architecture, etc.? Or should this be in the CONTRIBUTING.md?

Reply:
Agreed in the TC meeting: We will try to put the information about the decision making in the GOVERNANCE.md file, which doesn't change often and needs a higher level of discussion and agreement to change. The CONTRIBUTING.md and possibly other files contain the more practical details, which will change much more often and don't need so much discussion. These files can point to the GOVERNANCE.md to reference the decision making process.